flavoUr/views.py

```diff
@@ -0,0 +1,11 @@
+from django.shortcuts import get_object_or_404, render_to_response
+
+from django.template import loader
+from django.template import RequestContext
+
+from django.http import HttpResponseRedirect, HttpResponse, Http404
+
```

Comment:
yeah i just wanted to separate the different objects. but i guess 3/4 can be the same line